Assets/Scripts/Scenes/InGame/Stage/StageLight.cs

```diff
@@ -10,18 +11,36 @@ public class StageLight : MonoBehaviour, IInteractable
         [SerializeField] private List<GameObject> _lightObject = new List<GameObject>();
         private float[] _lightStrength = new float[10];
         private bool _isOn = false;
+        private bool _isAnimation = false;
+        private Vector3 SWICH = new Vector3(0, -40, 0);
+        private BoxCollider _lightCollider;
+        private bool _initialStateOpen = true;
+        private GameObject[] _child;
 
         void Awake()
         {
+            _lightCollider = GetComponent<BoxCollider>();
+
             for (int i = 0; i < _lightObject.Count; i++)
             {
                 _lightStrength[i] = _lightObject[i].GetComponent<Light>().intensity;
             }
+            
+            _child = new GameObject[]{ transform.GetChild(1).gameObject, transform.GetChild(2).gameObject};
+
+            if (_initialStateOpen)
+            {
+                _lightCollider.isTrigger = false;
+                LightOn();
+                _isOn = true;
+            }
         }
         public void Intract(PlayerStatus status)
         {
             if (Input.GetMouseButtonDown(1))
             {
+                _lightCollider.isTrigger = true;
+                _isAnimation = true;
```

Comment:
```suggestion
                if (_isAnimation) return;
                _lightCollider.isTrigger = true;
                _isAnimation = true;
```
間違っていたら申し訳ないのですが、Animation中はswitchの処理を行わないための関数だと思います。
その場合は、_isAnimationがtrueの際は処理を止める必要があります。

Reply:
_isAnimationがtrueの際処理を止めるよう変更しました